### DISCUSSION TITLE
[@types/mocha] missing runAsync on Mocha and Mocha.Runner

### DISCUSSION THREAD

**Discussion by mkhl** [DefinitelyTyped/DefinitelyTyped#59250] [Issues with a @types package] [answered]
the programmatic mocha api supports `runAsync`, which returns a promise instead of calling a callback.

see https://github.com/mochajs/mocha/blob/master/lib/mocha.js#L1062 and https://github.com/mochajs/mocha/blob/master/lib/runner.js#L1144

[a comment by an automated account is not shown]

**Comment by nicojs** [comment, marked as answer]
It should first be added to the public API: https://mochajs.org/api/

Anything not in the public API is considered private

**Reply by mkhl** [reply]
good point. thanks!
